### PR TITLE
THIRDEYE-853 : Create migration script for new db tables anomaly_tasks and anomaly_jobs

### PR DESCRIPTION
**Pull request by npawar**
This pull request contains the changeset for anomaly_tasks and anomaly_jobs table

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/main/resources/migrations.xml [hunk 1]
```diff
@@ -363,4 +363,63 @@
         onDelete="CASCADE"
     />
   </changeSet>
+
+  <changeSet id="13" author="npawar">
+    <!-- AnomalyJobSpec -->
+    <createTable tableName="anomaly_jobs">
+      <column name="job_execution_id" type="bigint" autoIncrement="true">
```
Comment (puneetjaiswal) [inline thirdeye/thirdeye-pinot/src/main/resources/migrations.xml]: Can we call this just 'id' ?

File: thirdeye/thirdeye-pinot/src/main/resources/migrations.xml [hunk 2]
```diff
@@ -363,4 +363,63 @@
...
+        <constraints nullable="true"/>
+      </column>
+      <column name="window_end_time" type="bigint">
+        <constraints nullable="true"/>
+      </column>
+    </createTable>
+
+    <!-- AnomalyTaskSpec -->
+    <createTable tableName="anomaly_tasks">
+      <column name="task_id" type="bigint" autoIncrement="true">
```
Comment (puneetjaiswal) [inline thirdeye/thirdeye-pinot/src/main/resources/migrations.xml]: task_id ----> id ?

**Comment by puneetjaiswal** [comment]
Going forward we can have An abstract Entity class having id as a common variable.

**Comment by npawar** [comment]
Fixed issues

**Comment by puneetjaiswal** [comment]
LGTM. go ahead and merge after resolving the build issue.

**Comment by npawar** [comment]
Checked with @Jackie-Jiang that the build issue can be ignore. It is usually disabled for thirdeye commits, but was manually disabled right now.

**Pull request merged**